config/config.go

```diff
@@ -87,8 +87,9 @@ type PoliciesConfig struct {
 	//
 	// This option should only be used when moving an installation to a new database.
 	AllowExplicitPolicyID bool `json:"allow_explicit_policy_id"`
-	// This option is used for storing a policies  if `policies.policy_source` is set to `file`.
-	// it should be some existing file path on hard drive
+	// If `policies.policy_source` is set to `file` and `policies.policy_path` is set. Then Tyk will load policies from the
```

Comment:
what happens if it's set and the source isn't?

Reply:
Not sure what the default behaviour is for the policy source. I would assume it would be file.